operators/pkg/controller/elasticsearch/pod/pod.go

```diff
@@ -29,6 +31,25 @@ var (
 		{Name: "transport", ContainerPort: network.TransportPort, Protocol: corev1.ProtocolTCP},
 		{Name: "process-manager", ContainerPort: processmanager.DefaultPort, Protocol: corev1.ProtocolTCP},
 	}
+
+	// DefaultVolumeClaimsTemplates is the default volume claim templates for Elasticsearch pods
+	DefaultVolumeClaimsTemplates = []corev1.PersistentVolumeClaim{
+		{
+			ObjectMeta: v1.ObjectMeta{
+				Name: "elasticsearch-data",
+			},
+			Spec: corev1.PersistentVolumeClaimSpec{
+				AccessModes: []corev1.PersistentVolumeAccessMode{
+					corev1.ReadWriteOnce,
+				},
+				Resources: corev1.ResourceRequirements{
+					Requests: corev1.ResourceList{
+						corev1.ResourceStorage: resource.MustParse("1Gi"),
```

Comment:
I'm wondering if this is big enough as a default. Could we get into a situation where many users start with a simple copy/pasted spec, get their 1Gi storage full and have basically lost their data at this point?
I'm seeing [Helm Charts default to 30Gi](https://github.com/elastic/helm-charts/blob/51ab321e5640e282392087536c554e4bd4239fd2/elasticsearch/values.yaml#L76), maybe we should align here?
Not sure that would fit Minikube very well though :/

Reply:
1Gi works well with Minikube etc, and doesn't accidentally incur a large cost. It's probably best if users are conscious about this number in any production scenario.